packages/strapi-plugin-i18n/admin/src/hooks/useLocales/index.js

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useReducer } from 'react';
```

Comment:
Same about this hook. We should later create one, that will be used in the webhooks, and the roles list